acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/Eth.java

```diff
@@ -47,4 +48,15 @@ public String sendTransaction(final Transaction transaction) throws IOException
   public String getCode(final String address) throws IOException {
     return jsonRpc.ethGetCode(address, DefaultBlockParameterName.LATEST).send().getResult();
   }
+
+  public BigInteger getBalance(final String account) throws IOException {
```

Comment:
Why is this named getBalance? This isn't a getter and it's just called balance in Accounts

Reply:
it's a wrapper for eth.getBalance. It provides the syntax of `eth.getBalance` which mirrors `Geth_getBalance` from the Ethereum JSON-RPC 😉

Reply:
fair enough